conan/cli/commands/graph.py

```diff
@@ -156,6 +156,8 @@ def graph_info(conan_api, parser, subparser, *args):
         conan_api.graph.analyze_binaries(deps_graph, args.build, remotes=remotes, update=args.update,
                                          lockfile=lockfile)
         print_graph_packages(deps_graph)
+        if profile_host.conf.get("tools.system.package_manager:mode") == "collect":
```

Comment:
This is the first usage of such a conf in our code, getting it directly from profile instead of ``conanfile``, but I don't see a strong reason to not allow it in our codebase.